index.html

```diff
@@ -77,7 +77,6 @@
           <ul class="nav navbar-nav">
             <li class="active"><a href="#">Home</a></li>
             <li><a href="http://github.com/Samsung/iotjs">View on Github</a></li>
-            <li><a href="https://github.com/Samsung/iotjs/wiki/IoT.js-Package">ipm</a></li>
 			<li><a href="http://www.jerryscript.net">Powered by <b>JerryScript</b></a></li>
```

Comment:
A bit nitpicky, but could you fix the indentation of this line if you touch the file anyway? :)